sherpa/astro/fake.py

```diff
@@ -143,9 +148,9 @@ def fake_pha(data, model,
     rmf0 = data.get_rmf(data.response_ids[0])
     data.channel = np.arange(1, rmf0.detchans + 1)
 
-    # Calculate the source model, and take a Poisson draw based on
+    # Calculate the source model, and take a "draw" based on
```

Comment:
why 'Poisson draw' was changed here? What is the distribution for the assumed draw otherwise?

Reply:
Because we can now pass in a function to do this, so it need not be a poisson distribution. The obvious choice is the identity function (ie adds no noise) as that is a good way to check the results,  but this is only when you want to check it's doing the "right" thing. I expect essentially everyone to use the default (which is poisson)

Reply:
That would be a good example in our docs we should add later.